lib/cmd/lint.js

```diff
@@ -121,6 +154,13 @@ function checkComponent(url, prefix, concurrency) {
  * @return {Stream}
  */
 function checkPage(url, prefix, concurrency) {
+  let ext = '';
+
+  // if we're checking a page with an extension, cut it off from the url and pass it into the component checking
```

Comment:
i don' think clay supports non-html renderers for pages? might just update the comment to read `if we're checking a page with a .html extension [...]`?

Reply:
It can support any type of renderer, it's just a question of if one has been built yet.

Reply:
Clay supports a _lot_ of different renderers!

![screen shot 2018-07-05 at 1 00 17 pm](https://user-images.githubusercontent.com/447522/42337011-6fbaa23e-8053-11e8-93b2-8f12a283f768.png)

Reply:
oh! 👌